src/config.hpp.in

```diff
@@ -29,5 +29,6 @@
 #cmakedefine ENABLE_MKLGPU_BACKEND
 #cmakedefine ENABLE_NETLIB_BACKEND
 #cmakedefine BUILD_SHARED_LIBS
+#cmakedefine REF_BLAS_LIBNAME "@BLAS_file@"
```

Comment:
So, with this change, how can the user control the REF_BLAS_LIBNAME? Should this information be added to README?

Reply:
REF_BLAS_LIBNAME would only be set by cmake once it found the libblas.{so,dll} file, the user could manually change build>bin>oneapi>mkl>detail>config.hpp to update the `define REF_BLAS_LIBNAME` line after cmake configuration step if they needed.

config.hpp after configuration step:
```
#define ENABLE_MKLGPU_BACKEND
/* #undef ENABLE_NETLIB_BACKEND */
#define BUILD_SHARED_LIBS
#define REF_BLAS_LIBNAME "/project/ahentabl/3.9.1/blas/lnx/lib/libblas.so"
```

Reply:
Also, for windows build config.hpp looks like:
```
#define ENABLE_MKLGPU_BACKEND
/* #undef ENABLE_NETLIB_BACKEND */
#define BUILD_SHARED_LIBS
#define REF_BLAS_LIBNAME "C:/3.9.1/blas/win/lib/blas.lib"
```
will look into why this is referencing blas.lib and not blas.dll

Reply:
This may be tricky. Try searching FindCBLAS.cmake for a different variable. If there isn't one, then you will have to check `IMPORTED_NAME` or similar properties on the `blas.lib` target.